third-party/json_fwd.hpp

```diff
@@ -0,0 +1,198 @@
+//     __ _____ _____ _____
```

Comment:
What is the purpose of this new file?

Reply:
Forward decl file, same as `<iosfwd>` for example